packages/README.md

```diff
@@ -0,0 +1,6 @@
+AWS CDK8s
```

Comment:
why do we need a readme file here?

Reply:
Strictly speaking we don't. Traditionally, it's here as a fallback in case a package doesn't have a README of it's own, then this gets plugged in. But I can get rid of it since it doesn't look like we need that functionality.

Reply:
LEt's